lib/converters/converter-v2-to-v1.js

```diff
@@ -334,6 +334,12 @@ _.extend(Builders.prototype, {
         _.map(units, function (unit) {
             request[unit] = self[unit](item);
         });
+
+        // description transformations for v2 to v1
+        request.queryParams = item.request.url.query;
```

Comment:
v2 might contain URLs as strings, make sure you parse the URL to an object before doing this.